l10n_th_withholding_tax_cert_multi/wizard/create_withholding_tax_cert.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2020 Ecosoft Co., Ltd (https://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+INCOME_TAX_FORM = [
+    ("pnd1", "PND1"),
+    ("pnd3", "PND3"),
+    ("pnd3a", "PND3a"),
+    ("pnd53", "PND53"),
+]
+
+WHT_CERT_INCOME_TYPE = [
```

Comment:
This too, should import from existing module.

Reply:
Edited.